packages/block-library/src/comments-pagination/edit.js

```diff
@@ -73,14 +76,28 @@ export default function QueryPaginationEdit( {
 		<>
 			{ hasNextPreviousBlocks && (
 				<InspectorControls>
-					<PanelBody title={ __( 'Settings' ) }>
-						<CommentsPaginationArrowControls
-							value={ paginationArrow }
-							onChange={ ( value ) => {
-								setAttributes( { paginationArrow: value } );
-							} }
-						/>
-					</PanelBody>
+					<ToolsPanel
+						label={ __( 'Settings' ) }
+						resetAll={ () =>
+							setAttributes( { paginationArrow: 'none' } )
+						}
+					>
+						<ToolsPanelItem
+							label={ __( 'Arrow Controls' ) }
```

Comment:
```suggestion
							label={ __( 'Arrow' ) }
```

This menu item text should match the label of the UI as much as possible.

Reply:
- [x] Updated the ToolsPabelItem Label from `Arrow Controls` to `Arrow`
- [x] Added dropdownMenuProps